api-specifications.md

```diff
@@ -300,6 +291,7 @@ list_only=/* true or false */
 * Assignment not found
 * Student not found
 * Submission not found
+* get_all, get_latest, and timestamp are mutually exclusive
```

Comment:
@Lawrence37 Can you put this line to about line 264 (in API description) and change error message to "Please specify exactly one of get_all, ..."?